src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -955,7 +965,7 @@ private void VerifyCoresPerNode(Dictionary<string, DetailsPerNode> detailsPerNod
 
             if (licenseChanged && _licenseStatus.MaxCores > _previousLicenseCores)
             {
-                changedNodes = detailsPerNode.Select(x => x.Key).ToList();
+                changedNodes = detailsPerNode.Where(x=>x.Value.Modified == false).Select(x => x.Key).ToList();
```

Comment:
Shouldn't this be `.Where(x => x.Value.Modified)` ?

Reply:
No, Modified means that the number of utilized cores, was set by the user explicitly.
We only want to reassign cores that wasn't set by user.